images/gcloud/Dockerfile

```diff
@@ -18,9 +18,12 @@ MAINTAINER senlu@google.com
 
 RUN apt-get update && apt-get install -y \
     python \
+    python-pip \
     wget && \
     apt-get clean
 
+RUN pip install requests
```

Comment:
Are we ok downloading latest version of requests as of the time the image is built? (or do we want to pin specific versions of requests)?

Reply:
seems other places like https://github.com/kubernetes/test-infra/blob/b6bb342cb0585ccf207aa65718990c628d4f9386/verify/test_requirements.txt#L3 does not pin version for requests. And we'd probably convert all those to bazel rules in the near future.